client/main.lua

```diff
@@ -779,3 +779,75 @@ RegisterNUICallback('engine', function()
     ToggleEngine(GetVehicle())
     SetNuiFocus(false, false)
 end)
+
+
+-------------------------------------
+-- TURN OFF VEHICLE WHEN U WANT TOGGLE WITH /startengine OR "G" or any BINDED TO THAT FUNCTION BUTTON. --
+-------------------------------------
+local engineManuallyToggled = {}
+
+
+CreateThread(function()
+    while true do
+        Wait(0)
+        local ped = PlayerPedId()
+        if IsPedInAnyVehicle(ped, false) then
+            local veh = GetVehiclePedIsIn(ped, false)
+            local plate = QBCore.Functions.GetPlate(veh)
+
+            if GetPedInVehicleSeat(veh, -1) == ped then
+                if not engineManuallyToggled[plate] then
+                    
+                    if GetIsVehicleEngineRunning(veh) then
+                        SetVehicleEngineOn(veh, false, true, true)
+                    end
+
+                    
+                    local speed = GetEntitySpeed(veh)
+                    
+                    DisableControlAction(0, 71, true) -- Accelerate (W)
+                    DisableControlAction(0, 86, true) -- Horn (E)
+                    
+                    
+                    if speed < 1.0 then
+                        DisableControlAction(0, 72, true) -- Brake (S)
+                    end
+
+                    DisableControlAction(0, 63, true) -- Steer Left (A)
+                    DisableControlAction(0, 64, true) -- Steer Right (D)
+                end
+            end
+        else
+            Wait(500)
+        end
+    end
+end)
```

Comment:
This most likely does not need to / shouldnt be ran every single frame of being in a vehicle.
Realistically, this should just be ran once upon entering a vehicle.

Nor should we really need to disable controls for this, as that will be done by disabling/turning off the engine.

Looking at the rest of this code, we can probably refactor this to a function that gets ran once upon `QBCore:Client:VehicleInfo` being triggered.